app/src/test/java/org/glucosio/android/tools/ReadingToolsTest.java

```diff
@@ -0,0 +1,32 @@
+package org.glucosio.android.tools;
+
+import org.glucosio.android.RobolectricTest;
+import org.junit.Test;
+
+import java.util.Locale;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReadingToolsTest extends RobolectricTest {
+    @Test
+    public void shouldReturnNull_whenNullStringPassed() {
+        assertThat(ReadingTools.parseReading(null)).isNull();
+    }
+
+    @Test
+    public void shouldReturnNull_whenInvalidStringPassed() {
+        assertThat(ReadingTools.parseReading("abc")).isNull();
+    }
+
+    @Test
+    public void shouldReturnCorrectNumber_whenDotIsDecimalSeparator() {
+        Locale.setDefault(new Locale("en"));
```

Comment:
That would influence all following tests in the suite. Should we return original local in `@After`?